packages/plugins/test/mocha/common.ts

```diff
@@ -339,53 +340,64 @@ export const makeDeployment = async (
   const networkNames = mainnets.concat(testnets)
 
   const collectedPromises = networkNames.map(async (networkName) => {
-    const chainId = fetchChainIdForNetwork(networkName)
```

Comment:
The diff below looks messy, but I just wrapped this code block in a try/catch so that we can identify which network caused an error (if one occurs)